orbit/cmd/desktop/desktop.go

```diff
@@ -433,6 +435,42 @@ func main() {
 	systray.Run(onReady, onExit)
 }
 
+func refreshBubble(sum fleet.DesktopSummary, selfServiceItem *systray.MenuItem, myDeviceItem *systray.MenuItem) {
+	// Check for null for backward compatibility with an old Fleet server
+	if sum.SelfService != nil && !*sum.SelfService {
+		selfServiceItem.Disable()
+		selfServiceItem.Hide()
+	} else {
+		selfServiceItem.Enable()
+		selfServiceItem.Show()
+	}
+
+	failingPolicies := 0
+	if sum.FailingPolicies != nil {
+		failingPolicies = int(*sum.FailingPolicies)
+	}
+
+	if failingPolicies > 0 {
+		if runtime.GOOS == "windows" {
+			// Windows (or maybe just the systray library?) doesn't support color emoji
+			// in the system tray menu, so we use text as an alternative.
+			if failingPolicies == 1 {
+				myDeviceItem.SetTitle("My device (1 issue)")
+			} else {
+				myDeviceItem.SetTitle(fmt.Sprintf("My device (%d issues)", failingPolicies))
+			}
+		} else {
+			myDeviceItem.SetTitle(fmt.Sprintf("🔴 My device (%d)", failingPolicies))
+		}
+	} else {
+		if runtime.GOOS == "windows" {
+			myDeviceItem.SetTitle("My device")
+		} else {
+			myDeviceItem.SetTitle("🟢 My device")
```

Comment:
Unrelated comment: A few years ago we added this green bubble emoji because we couldn't figure out changing the Fleet icon to a Fleet icon with a ! or something indicate failure. Currently the user needs to click on the Fleet icon to realize some policy is failing. 

Maybe we can fix this some day given we recently changed the underlying package from https://github.com/getlantern/systray to https://github.com/fyne-io/systray? :)